modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceReplacementProducts.testcase

```diff
@@ -184,6 +184,109 @@ definition {
 		}
 	}
 
+	@description = "COMMERCE-12579. As a buyer, I can view the discontinued SKU shown in the product's detail replacement tab"
+	@priority = 3
+	test CanReplacementSkusShownInProductDetails {
+		property portal.acceptance = "false";
+
+		task ("Given 5 products are set as Discontinued with a same replacement product ABS Sensor") {
+			for (var entryList : list "Brake Fluid:MIN93016A,Brake Fluid:MIN93016B,Brake Fluid:MIN93016C,Wear Sensors:MIN93027,Calipers:MIN93021") {
+				var productName = StringUtil.extractFirst(${entryList}, ":");
+				var sku = StringUtil.extractLast(${entryList}, ":");
+
+				CommerceProducts.openProductsAdmin();
+
+				CommerceProducts.searchAndGoToProductEntry(entryProductName = ${productName});
+
+				CommerceEntry.gotoMenuTab(menuTab = "SKUs");
+
+				CommerceNavigator.gotoEntry(entryName = ${sku});
+
+				SelectFrame(locator1 = "IFrame#IFRAME");
+
+				CommerceProducts.discontinueSKU();
+
+				CommerceProducts.setReplacementSKU(replacementSKU = "MIN93015");
+			}
+		}
+
+		task ("When the buyer navigates to ABS Sensor's detail page") {
+			User.logoutPG();
+
+			CommerceLogin.miniumLogin(
+				password = "test",
+				urlAppend = "web/minium",
+				userEmailAddress = "buyer@liferay.com");
+
+			Navigator.openWithAppendToBaseURL(urlAppend = "web/minium/p/abs-sensor");
+		}
+
+		task ("Then assert that the Replacements button shown") {
```

Comment:
Is there a button or tab for replacement products?

"Then can assert that the Replacements tab is shown"